lib/vfscore/README.md

```diff
@@ -0,0 +1,148 @@
+# VFSCore: Unikraft's Virtual File System API
+
+`vfscore` (*Virtual File System Core*) is an internal library of Unikraft that provides a generic interface of system calls related to filesystem management.
+There are multiple types of filesystems available in Unikraft (e.g: `ramfs`, `9pfs`).
+
+## Organization
+
+The `vfscore` library consists of multiple files:
+
+```console
+$ tree -L 1 lib/vfscore/
+lib/vfscore/
+[...]
+├── fd.c
+├── file.c
+├── fops.c
+├── include
+├── lookup.c
+├── main.c
+├── stdio.c
+├── syscalls.c
+├── task.c
+├── vfs.h
+└── vnode.c
+```
+
+- `vnode.c` implements functions that are used to interact with the `fdtable`
```

Comment:
I think `functions that interact with a vnode structure` is more suited. You can also point to the `vnode` strcuture in `include/vfscore/vnode.h`.